corehq/apps/reports/standard/project_health.py

```diff
@@ -327,6 +329,49 @@ def previous_six_months(self):
             last_month_summary = this_month_summary
         return six_month_summary
 
+    def export_summary(self, six_months):
+        table = []
+        table.append('Six Month Performance Summary')
+        worksheet_headers = ['month', 'num_high_performing_users', 'num_low_performing_users',
+                             'total_active', 'total_inactive', 'total_num_users']
+        table.append([worksheet_headers])
+        for month_summary in six_months:
+            table[1].append([month_summary.month.isoformat(), month_summary.number_of_performing_users,
+                             month_summary.number_of_low_performing_users, month_summary.active,
+                             month_summary.inactive, month_summary.total_users_by_month])
+        return table
```

Comment:
it's a little hard to understand the structure of this list - it looks like it's
```python
table = [
    "Six Month Performance Summary",
    [
        ['month', 'num_high_performing_users', ...],
        [...],
        [...],
    ]
]
```
Is that second element a sheet?

Might be clearer to do something like

```python
summary_sheet = [['month', 'num_high_performing_users', ...]]
for month_summary in six_months:
    summary_sheet.append([month_summary.isoformat()...])

return [
    "Six Month Performance Summary",
    [summary_sheet]
]
```
Basically using data literals more and minimizing index variables.